android/app/build.gradle

```diff
@@ -116,9 +116,13 @@ def enableHermes = project.ext.react.get("enableHermes", false)
  * Whether to enable building a seperate APK for each ABI.
  *
  * Defaults to false but can be set to true with the project properties flag
- * e.g. ./gradlew assembleDebug -Psplitapk=true
+ * e.g. ./gradlew assembleDebug -PsplitApk=true
+ *
+ * Additional option to include a universal APK
+ * e.g. ./gradlew assembleRelease -PsplitApk=true -PincludeUniversalApk=true
  */
-ext.splitapk = project.properties['splitapk'] ?: false
+ext.splitApk = project.properties['splitApk'] ?: false
+ext.includeUniversalApk = project.properties['includeUniversalApk'] ?: false
```

Comment:
Both properties to `false`? those values are by default?

Reply:
Both properties are false by default as this is the way the code was before
I added these options. I would like to keep it this way until we break open
the detox commands in case it causes unknown issues with e2e. Is that ok
with you?

Reply:
oh, yes. Approving...